extra-dev/packages/coq-autosubst/coq-autosubst.dev/opam

```diff
@@ -16,10 +16,10 @@ annotations, Autosubst synthesizes the parallel substitution
 operation and automatically proves the basic lemmas about
 substitutions."""
 
-build: [make "-j%{jobs}%"]
+build: [make "-j%{jobs}%" ]
```

Comment:
```suggestion
build: [make "-j%{jobs}%"]
```
If you regenerate the repo opam file with the latest [templates](https://github.com/coq-community/templates), this space will disappear.